.github/workflows/tests.yml

```diff
@@ -18,6 +18,8 @@ jobs:
             laravel: 'dev-master as 9.0'
           - php: '7.4'
             laravel: 'dev-master as 9.0'
+          - php: '8.0'
```

Comment:
Laravel 9.1 requires php 8.1 hence tests where failing

Reply:
Laravel 9 actually only requires PHP 8.0, what test failures did you see?

Reply:
Dev master is now 9.1 which requires 8.1, just trigger a travis build and you'll see

Reply:
https://github.com/laravel/framework/commit/c4271001aa15a34af41b8205674ce62cd6c7aaaf

Reply:
Ideally we could now just put 9.0 instead of dev master but it was just a quick fix to get the tests working again

Reply:
Ahh gotcha, that's right `dev-master` is technically for Laravel 10.x. I'll fix those tests in the base branch. Thanks!